modules/tablemetadata/model/tablemetadata.class.php

```diff
@@ -63,44 +65,64 @@ function updateallskipsystemdbs($args) {
 
     function runscriptall($args) {
         if (empty($args['scriptpath'])) {
-            print "Missing scriptpath argument.\n";
+            print "Missing scriptpath argument.<br>";
             return false;
         }
 
         $scriptpath = $args['scriptpath'];
 
         global $_lib;
 
-        $system_dbs = array('lodo', 'mysql', 'test', 'information_schema');
+        $system_dbs = $this->system_dbs;
 
         if (!is_file($scriptpath)) {
-            print "File $scriptpath not found.\n";
+            print "File $scriptpath not found.<br>";
             return false;
         }
 
         global $_SETUP;
 
         $script = file_get_contents($scriptpath);
 
-        
-        $params['commands'] = explode(';', $script);
+        // remove the home directory if there is any
+        $scriptpath = preg_replace('/.*db\/changes\//', 'db/changes/', $scriptpath);
+
+        // separate full path by / and reverse to get script_name as first element
+        $tmp = array_reverse(explode('/', $scriptpath));
+        // separate script_name by _ and get script_number as first element
+        $tmp = explode('_', $tmp[0]);
+        $script_number = (int) $tmp[0];
+        $commands = explode(';', $script);
+        // replace next two lines with the comented out ones below after the
+        // migration 133_create_migrations_table.sql is run
+        $commands_before = array();
+        $commands_after = array();
+        // $commands_before = array("REPLACE INTO migrations (MigrationID, MigrationName) VALUES ($script_number, '$scriptpath')");
```

Comment:
Set:
Status to STARTED,
SucceededAt to NULL,
StartedAt to NOW()
In order for us to be able to see the statuses of rerunnable migrations.